keps/sig-instrumentation/4785-crdmetrics-controller/kep.yaml

```
@@ -0,0 +1,38 @@
+title: CRDMetrics Controller
+kep-number: 4785
+authors:
+  - "@rexagod"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+status: implementable
```

As this PRs has TBD elements, I suggest for now:
```suggestion
status: provisional
```

---

[Right](https://github.com/rexagod/crdmetrics#todo), will do!